client/coral-framework/services/i18n.js

```diff
@@ -3,6 +3,11 @@ import has from 'lodash/has';
 import get from 'lodash/get';
 import merge from 'lodash/merge';
 
+import 'moment/locale/da';
+import 'moment/locale/es';
+import 'moment/locale/fr';
+import 'moment/locale/pt-br';
```

Comment:
Missing english locales :-P

Reply:
Nope haha! We needed to specify moment's locale. Done! 👍